src/EditorFeatures/CSharpTest/SymbolKey/SymbolKeyCompilationsTests.cs

```diff
@@ -458,7 +513,7 @@ public class Test
 {
     internal protected byte field = 255;    // change modifier and init-value
     internal string P { get { return null; } }       // remove 'set'
-    public int M(ref int n) { return 0;  }   // change ret type
+    public static void M(ref int n) { return 0;  }   // change to static
```

Comment:
This test is "just being cutesy to show that that works" so I changed how it was being cutesy since the old way is now supported as a non-cute option.